app/javascript/components/home/ProjectsComponent.tsx

```diff
@@ -10,7 +10,7 @@ interface Props {
 class ProjectsComponent extends React.Component<Props> {
   componentDidMount(): void {
     const { dispatch } = this.props;
-    dispatch(getProjects());
+    // dispatch(getProjects());
```

Comment:
this is 401'ing because when a page is rendered, it goes to the HomePage by default then is rerouted based on the request.

We need to change the routing mechanism and be okay with `localhost/#/route` I believe